virttest/utils_windows/drive.py

```diff
@@ -86,3 +86,23 @@ def shrink_volume(session, vol_id, size):
     shrink_cmd += 'echo shrink desired=%s >> {0} ' % size
     shrink_cmd += '&& diskpart /s {0}'
     session.cmd(shrink_cmd.format(script_path))
+
+
+def get_disk_property_by_serial_number(session, serial_number, property_name):
+    """
+    Get disk drive property value by serial number.
+
+    :param session: Windows VM session.
+    :type session: aexpect.ShellSession
+    :param serial_number: Serial number of disk drive.
+    :type serial_number: str
+    :param property_name: The name of property to be get.
```

Comment:
do we need add like "comma to sperate multi properties"

Reply:
Please check the latest code, using a `list` of multiple properties.